src/web_sys.rs

```diff
@@ -3370,6 +3370,24 @@ impl HasContext for Context {
         .unwrap_or(false)
     }
 
+    unsafe fn get_parameter_bool_array<const N: usize>(&self, parameter: u32) -> [bool; N] {
+        let value = match self.raw {
+            RawRenderingContext::WebGl1(ref gl) => gl.get_parameter(parameter),
+            RawRenderingContext::WebGl2(ref gl) => gl.get_parameter(parameter),
+        }
+        .unwrap();
+        use wasm_bindgen::JsCast;
+        let mut v = [false; N];
+        if let Some(value) = value.as_bool() {
```

Comment:
Is there an API that could return either of these types for one parameter? If not, I'd prefer to only handle `js_sys::Array` here, and expect people to use `get_parameter_bool` if GL says it will return a single `bool`.

Reply:
per https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/getParameter it should not

Reply:
done in https://github.com/grovesNL/glow/pull/308/commits/3d6e52ae4283210c33d7c077548e1b25edbc4007